pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -102,7 +102,6 @@ in stdenv.mkDerivation rec {
         glxinfo
         gnugrep
         gnused
-        xdg-utils
```

Comment:
The `./hardcode-dependencieis.patch` file will have to be updated to account for this change. I can take a look at this since I maintain mangohud.

Reply:
Hmm wait, yeah the reason I hard-code these dependencies is because the library includes it and it can be loaded independently from the executable.

I think for now we should just leave this unchanged, I'll try to figure out a better solution for another PR.